### PR TITLE
OSDOCS-7643: Privatelink endpoint creation

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7643

Link to docs preview:
https://64391--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access

QE review:
- [ ] QE has approved this change.
No QE review required at this stage. 

Additional information:
SME review on associated Google doc.

### CODE REVIEW

File: modules/rosa-sre-access-privatelink-vpc.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[id="rosa-sre-access-privatelink-vpc.adoc_{context}"]
+= SRE access through PrivateLink VPC endpoint service 
+
+PrivateLink VPC endpoint service is created as part of the ROSA cluster creation. 
+
+When you have a PrivateLink ROSA cluster, its Kubernetes API Server is exposed through a load balancer that can only be accessed from within the VPC by default. Red Hat site reliability engineering (SRE) can connect to this load balancer through a VPC Endpoint Service that has an associated VPC Endpoint in a Red Hat owned AWS account. This endpoint service contains the name of the cluster, which is also in the ARN.
```
Comment (EricPonvelle) [inline modules/rosa-sre-access-privatelink-vpc.adoc:6]: Minor suggestion:

s/a Red Hat owned AWS account/a Red Hat-owned AWS account

File: modules/rosa-sre-access-privatelink-vpc.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Under the *Allow principals* tab, a Red Hat owned AWS account is listed. This specific user ensures that other entities cannot create VPC Endpoint connections to the PrivateLink cluster’s Kubernetes API Server.
```
Comment (EricPonvelle) [inline modules/rosa-sre-access-privatelink-vpc.adoc:8]: If you make the previous change, mirror it here.

File: modules/rosa-sre-access-privatelink-vpc.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[id="rosa-sre-access-privatelink-vpc.adoc_{context}"]
+= SRE access through PrivateLink VPC endpoint service 
+
+PrivateLink VPC endpoint service is created as part of the ROSA cluster creation. 
+
+When you have a PrivateLink ROSA cluster, its Kubernetes API Server is exposed through a load balancer that can only be accessed from within the VPC by default. Red Hat site reliability engineering (SRE) can connect to this load balancer through a VPC Endpoint Service that has an associated VPC Endpoint in a Red Hat owned AWS account. This endpoint service contains the name of the cluster, which is also in the ARN.
+
+Under the *Allow principals* tab, a Red Hat owned AWS account is listed. This specific user ensures that other entities cannot create VPC Endpoint connections to the PrivateLink cluster’s Kubernetes API Server. 
+
+When Red Hat SREs access the API, this fleet management plane can connect to the internal API through the VPC endpoint service.
```
Comment (EricPonvelle) [inline modules/rosa-sre-access-privatelink-vpc.adoc:10]: "This fleet management plane" seems confusing. I didn't see the term "fleet management plane in this document before this. Maybe just "The fleet..."

**Review by EricPonvelle — COMMENTED** [review]
Small items but looks good otherwise

File: modules/rosa-sre-access-privatelink-vpc.adoc (hunk 3 above)
Comment (bmcelvee) [reply modules/rosa-sre-access-privatelink-vpc.adoc:10]: Agreed, but that was specific language provided by SRE, so I'm going to keep it as is

File: _topic_maps/_topic_map_rosa.yml [hunk 4]
```diff
@@ -67,8 +67,8 @@ Topics:
     File: rosa-life-cycle
   - Name: Understanding security for ROSA
     File: rosa-policy-process-security
-#  - Name: SRE and service account access
-#    File: rosa-sre-access
+  - Name: SRE and service account access
+    File: rosa-sre-access
```
Comment (bmcelvee) [inline _topic_maps/_topic_map_rosa.yml:71]: Reminder: comment out before merge.

**Pull request merged**